aspnetcore/host-and-deploy/azure-apps/index.md

```diff
@@ -202,6 +202,13 @@ When deploying a self-contained app:
 
 The [Docker Hub](https://hub.docker.com/r/microsoft/aspnetcore/) contains the latest preview Docker images. The images can be used as a base image. Use the image and deploy to Web Apps for Containers normally.
 
+## Protocol settings (HTTPS)
+
+Secure protocol bindings allow you specify which certificate to use when responding to requests to a specific hostname over HTTPS. Binding requires a valid private certificate (*.pfx*) issued for the specific hostname. For more information, see the following topics:
+
+* [Tutorial: Bind an existing custom SSL certificate to Azure Web Apps](/azure/app-service/app-service-web-tutorial-custom-ssl)
+* [How To Configure TLS Mutual Authentication for Web App](/azure/app-service/app-service-web-configure-tls-mutual-auth)
```

Comment:
Why this link? I understand TLS is a pre-req for mTLS but it does more to distract than help in this article

Reply:
They put this link at the end of their **SSL bindings** description.

![addbinding](https://user-images.githubusercontent.com/1622880/47685478-a25e2b00-dba4-11e8-9ca8-1111d63aebe2.png)

How about this one ... https://docs.microsoft.com/azure/app-service/web-sites-purchase-ssl-web-site ... or shall I just remove the 2nd link.

Reply:
My vote is to remove the link.